.travis.yml

```diff
@@ -1,14 +1,15 @@
 sudo: false
 language: go
 go:
-  - tip
+  - master
+  - 1.9
+  - 1.8
   - 1.7
   - 1.6
 before_script:
```

Comment:
With the release of Go 1.10, I'd recommend dropping support for Go 1.8 and below. Then you can remove the ugly `grep -v /vendor/` from `go test ./... | grep -v /vendor/`, etc. in `test.sh`.